Sources/CodeScanner/ScannerViewController.swift

```diff
@@ -405,11 +405,11 @@ extension CodeScannerView {
             lastTime = Date()
         }
 
-        func isPastScanInterval() -> Bool {
+        var isPastScanInterval: Bool {
             Date().timeIntervalSince(lastTime) >= parentView.scanInterval
         }
         
-        func isWithinManualCaptureInterval() -> Bool {
+        var isWithinManualCaptureInterval: Bool {
             Date().timeIntervalSince(lastTime) <= 0.5
         }
```

Comment:
That might be a breaking change since it's no longer a function (I saw when you renamed for example `require` to `requires`, you kept it available but deprecated; here it breaks since it's the same name but with different type)

Reply:
Yes, you are right. I will add deprecated versions.